app/boards/arm/bdn9/bdn9_rev2.yaml

```diff
@@ -12,3 +12,6 @@ supported:
   - switches
   - underglow
   - per_key
+manufacturer: keeb.io
```

Comment:
If the url is mandatory then the manufacturer field seems superfluous (and arguably ambiguous), especially if it isn't displayed anywhere.

Reply:
It seems like a rather important detail to leave out for a hypothetical future database full of these hardware pieces. It offers a common property between hardware pieces created by the same entity.

Reply:
If so, a KV lookup (manufacturer id to display name) would be more appropriate, less error prone and more extensible.  So if we're not displaying it yet, we might as well leave it out until later (I've already extracted a copy to sheets if you want to save it for said future efforts).